### PR TITLE
SEO-3239 ASP.NET MVC Page with redirect

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @Aishwarya-Ganesan ,

Please review the changes for ASP.NET MVC Page with redirect.

Regards,
Yvone.

### CODE REVIEW

File: aspnetmvc/Grid/How-to.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-title: How to
-description: How to
+title: How to Section in ASP.NET MVC Grid control | Syncfusion
+description: Learn here more about How To Section in Syncfusion Essential ASP.NET MVC Grid Control, its element and more.
 platform: ejmvc
 control: Grid
 documentation: ug
 ---
-# How to
+# How to Section in ASP.NET MVC
```
Comment (gowthamanthavasiyappan) [inline aspnetmvc/Grid/How-to.md:9]: Add Control name in H1 tag

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]

**Comment by gowthamanthavasiyappan** [comment]
Commented

File: aspnetmvc/Grid/How-to.md (hunk 1 above)
Comment (Yvone-Atieno) [reply aspnetmvc/Grid/How-to.md:9]: added control name

**Review by gowthamanthavasiyappan — APPROVED** [review]

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**